spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile2.1]]
+=== MicroProfile 2.1 === (4Q2018)
+MicroProfile 2.1 is the eigth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+Based on MicroProfile's time-boxed release process, this is an incremental release that includes an update to OpenTracing 1.2.
+
```

Comment:
I would do a hotlink to OpenTracing 1.2, just to make it easy for readers to find the updated component.

Reply:
Resolved.